README.md

```diff
@@ -1,2 +1,159 @@
-# My-portfolio-site
-I am going to use this repo for building my portfolio site(Mobile First). 
+<a name="readme-top"></a>
+
+<div align="center">
+  <br/>
+
+  <h3><b>My-Portfolio_site</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [My-Portfolio-site] <a name="about-project"></a>
+
+This project is about building a personal portfolio website - specifically about the mobile first usage. By giving a glance to this project you can understand the usage of flexbox container and flexbox items. The purpose of this project is making an effective, SEO friendly, responsive, and quality personal portfolio to showcase myself to the professional market.  
+
+## 🛠 Built With <a name="built-with"></a>
+-HTML5
+-CSS3
+
+### Tech Stack <a name="tech-stack"></a>
+<details>
+  <summary>Version Control</summary>
+  <ul>
+    <li><a href="https://github.com/">Git Hub</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Visual Studio Code</summary>
+  <ul>
+    <li><a href="https://code.visualstudio.com">Visual Studio Code</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com/html5/">HTML5</a></li>
+    <li><a href="https://www.css3.com/">CSS</a></li>
+  </ul>
+</details>
+ 
+### Key Features <a name="key-features"></a>
+
+- **[Usage of flexbox]**
+- **[Mobile responsives portfolio setup]**
+- **[Use-of-HTML5 tools]**
+- **[Follow HTML and CSS best practice]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+-Visual Studio Code
+-Github Account
+-Git Bash
+-Node JS
+-Figma Templates
+
+### Setup
+
+Clone this repository to your desired folder:
+Use git clone command or downloaded ZIP folder
+
+### Install
+
+Install this project with:
+npm
+
+### Usage
+
+To run the project, execute the following command:
+live server or npm start
```

Comment:
- [ ] Kindly provide enough information for the setup, installation, and the usage of your web app. Make sure you write all the `npm commands` you used :+1: